<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateAttachClientsForAreaDialog</class>
 <widget class="QDialog" name="CreateAttachClientsForAreaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1215</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выполнить прикрепление пациентов к участкам</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="text">
        <string>Прикрепить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPrint">
       <property name="text">
        <string>Печать</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="childrenCollapsible">
         <bool>false</bool>
        </property>
        <widget class="CClientAttachMonitoring" name="clientAttachMonitoring" native="true">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="margin">
           <number>0</number>
          </property>
         </layout>
        </widget>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="1" column="0">
           <widget class="QProgressBar" name="progressBar">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QTextBrowser" name="logBrowser"/>
          </item>
          <item row="0" column="0">
           <widget class="QFrame" name="frmFilters">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <layout class="QGridLayout" name="gridAttachByAddress">
               <item row="2" column="0">
                <widget class="QLabel" name="lblAreaAddressType">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Участок по адресу</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="lblOrgStructure">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Подразделение</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="lblAttachBy">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="toolTip">
                  <string>&lt;i&gt;По сетям, затем по типу участка&lt;/i&gt; — при выборе этого варианта программа пытается найти подходящий участок по параметрам его сети, если это не удалось, пытается найти терапевтический участок для взрослого клиента и педиатрический для ребенка. Если и это не удалось, то пытается прикрепить к участку общей практики.&lt;br /&gt;&lt;br /&gt; &lt;i&gt;Только по сетям&lt;/i&gt; — программа будет подбирать участок только по сети. &lt;br /&gt;&lt;br /&gt;&lt;i&gt;Только по типу участка&lt;/i&gt; — программа подбирает подходящий участок сразу по типу участка.</string>
                 </property>
                 <property name="text">
                  <string>Осуществлять прикрепление</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QComboBox" name="cmbSex">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <item>
                  <property name="text">
                   <string>Все</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Мужчины</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Женщины</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="lblSex">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Пол</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QComboBox" name="cmbAreaAddressType">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <item>
                  <property name="text">
                   <string>Регистрация</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Проживание</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="lblAge">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Возраст</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QComboBox" name="cmbAge">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <item>
                  <property name="text">
                   <string>Все</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Дети</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Взрослые</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="6" column="1">
                <widget class="QComboBox" name="cmbAttachBy">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <item>
                  <property name="text">
                   <string>По сетям, затем по типу участка</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Только по сетям</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Только по типу участка</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="COrgStructureComboBox" name="cmbOrgStructure">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QCheckBox" name="chkAttach">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::RightToLeft</enum>
                 </property>
                 <property name="text">
                  <string>Учитывать тип прикрепления:  Прикрепление</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" colspan="2">
                <widget class="QRadioButton" name="rbtnByAddress">
                 <property name="text">
                  <string>По адресу</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroup</string>
                 </attribute>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="Line" name="line">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridMoveFromOneToOther">
               <item row="0" column="0" colspan="2">
                <widget class="QRadioButton" name="rbtnFromOneToOther">
                 <property name="text">
                  <string>Без учета адреса</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroup</string>
                 </attribute>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Кол-во</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Preferred</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="lblSex2">
                 <property name="text">
                  <string>Пол</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QComboBox" name="cmbAge2">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <item>
                  <property name="text">
                   <string>Все</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Дети</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Взрослые</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="lblAge2">
                 <property name="text">
                  <string>Возраст</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="cmbSex2">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <item>
                  <property name="text">
                   <string>Все</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Мужчины</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Женщины</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="3" column="0" colspan="2">
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="lblFrom">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>с</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="COrgStructureComboBox" name="cmbFromOrgStructure">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="lblTo">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>на</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="COrgStructureComboBox" name="cmbToOrgStructure">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="4" column="1">
                <widget class="QSpinBox" name="edtAttachedCount">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="maximum">
                  <number>100000</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CClientAttachMonitoring</class>
   <extends>QWidget</extends>
   <header location="global">Orgs/ClientAttachMonitoring.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>rbtnByAddress</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbOrgStructure</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbtnByAddress</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbAreaAddressType</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbtnByAddress</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkAttach</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbtnByAddress</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbSex</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbtnByAddress</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbAge</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>407</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbtnByAddress</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbAttachBy</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbtnFromOneToOther</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbSex2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>684</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>984</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbtnFromOneToOther</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbAge2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>665</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>690</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbtnFromOneToOther</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbFromOrgStructure</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>763</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbtnFromOneToOther</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbToOrgStructure</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>944</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>1048</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbtnFromOneToOther</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtAttachedCount</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>691</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
